host/src/FunctionsNetHost.Shared/EnvironmentVariables.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace FunctionsNetHost.Shared
+{
+    public static class EnvironmentVariables
+    {
+        /// <summary>
+        /// The environment variable which is used to specify the specialized (function app payload) entry assembly.
+        /// </summary>
+        public const string SpecializedEntryAssembly = "AZURE_FUNCTIONS_NETHOST_SPECIALIZED_ENTRY_ASSEMBLY";
+
+        /// <summary>
+        /// The environment variable which is used to specify the path to the jittrace file which will be used for prejitting.
+        /// </summary>
+        public const string PreJitFilePath = "AZURE_FUNCTIONS_FUNCTIONSNETHOST_PREJIT_FILE_PATH";
```

Comment:
Update all instances of "AZURE_FUNCTIONS_FUNCTIONSNETHOST_" to match"

```suggestion
        public const string PreJitFilePath = "AZURE_FUNCTIONS_NETHOST_PREJIT_FILE_PATH";
```

Reply:
Changed all occurrences. I will update the WIKI where these env variables are documented.